about.html

```diff
@@ -0,0 +1,178 @@
+                />
+              </svg>
+            </a>
+  
+            <span class="span">Logout</span>
+            <span class="span">MyPage</span>
+            <span class="span">English</span>
+          </div>
+        </header>
+    </nav>
+
+    <div class="container-fluid"></div>
+        <div class="navdiv sticky-top"></div>
+          <nav class="navbar d-none d-md-flex maxwidth ">
+             <a href="index.html"><img class="navbar-brand" src="./Images/Spectra-logo.webp" alt="Spectra-logo logo"> </a> 
+             <div class="navlinks d-md-flex justify-content-between align-items-center">          
+               <a href="index.html">Home</a>
+               <a href="#" class="active">About</a>
+               <a href="Services">Services</a>
+               <a href="#" class="nav-link disabled">Contact</a> 
+               <button type="button">Book Now</button>  
+             </div>
+           </nav>
+        
+           <div class="menu">
+
+			<div id="myNav" class="overlay">
+
+				<a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+
+				<div class="overlay-content">
+					<a href="index.html" onclick="closeNav()">Home</a>
+					<a href="#" onclick="closeNav()">About</a>
+					<a href="#" onclick="closeNav()">Services</a>
+                    <a href="#" onclick="closeNav()">Contact</a>
+				</div>
+			</div>
+           <nav class="d-flex justify-content-between d-md-none maxwidth smallnav pl-3 navbar ">        
+            <a href="index.html"><img class="navbar-brand-img1" src="./Images/Spectra-logo.webp" alt="Spectra logo"></a>         
+            <span  onclick="openNav()" class="fas fa-bars" data-toggle="collapse" data-target="#collapsibleNavbar"></span>
+            <div class="collapse navbar-collapse py-2" id="collapsibleNavbar">
+                <ul class="navbar-nav pl-3 float-right bg-light">
+                    <li class="navlinks "><a href="index.html">Home</a></li>
+                    <li class="navlinks"><a href="about.html">About</a></li>
+                    <li class="navlinks"><a  href="Services">Services</a></li>
+                </ul>
+            </div>          
+        </nav>
+     </div>  
+
+     <section class="about">
+        <div class="about-content">
+            <h1>Welcome to Spectra Beauté</h1>
+          <h2 class="message">"We believe in our responsibility to inspire"</h2>
+        </div>
+        <div class="Description">
+          <p>
+            Spectra Beauté is considered by most as the leading beauty and wellness salon/spa in Cameroon (and arguably Central Africa). A source of inspiration and wellbeing and my love for this brand is embedded through this creative web-design and the joy, peace i feel when using its exceptional and unique products.
+          </p>
```

Comment:
- Kindly fix the overlay on the mobile screen for this description 👍🏽 
![screenshot](https://user-images.githubusercontent.com/44624138/181951741-0f3e980b-9658-4a41-9c3e-5641d3e4690f.png)